### DISCUSSION TITLE
0.23.0-rc2: OTP - error when I try updating duration and password length

### DISCUSSION THREAD

**Discussion by MichaelSowah** [pocketbase/pocketbase#5633] [Q&A] [answered]
When I try to update the OTP duration and generated password length from the admin UI, I get the error "Failed to load data due to invalid formatting."

Kindly assist.

**Comment by ganigeorgiev** [comment, marked as answer]
Please use `v0.23.0-rc3`. See also the complete release notes - https://github.com/pocketbase/pocketbase/releases.
